### PR TITLE
Cleanup tests code in preparation for Hats submissions.

### PR DESCRIPTION
**Pull request by deuszx**

### CODE REVIEW

File: amm/drink-tests/src/utils.rs [hunk 1]
```diff
@@ -45,104 +48,130 @@ pub fn upload_all(session: &mut Session<MinimalRuntime>) {
...
+
+    /// Returns balance of given token for given account.
+    /// Fails if anything other than success.
+    pub fn balance_of(
+        session: &mut Session<MinimalRuntime>,
+        token: AccountId,
+        account: AccountId,
+    ) -> u128 {
+        session
+            .query(PSP22::balance_of(&token.into(), account))
```
Comment (krzysztofziobro) [inline amm/drink-tests/src/utils.rs:162]: Can we use `handle_ink_error` in this file also?

File: .github/workflows/pr-checks.yaml [hunk 2]
```diff
@@ -24,5 +24,5 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-rust
 
-    - name: Rust checks
-      run: make check-all-dockerized
+    - name: DRink! tests
+      run: make all-drink-dockerized
```
Comment (krzysztofziobro) [inline .github/workflows/pr-checks.yaml:28]: Does it mean that CI won't run unit tests now, or am I mistaken?

**Review by DamianStraszak — APPROVED** [review]

File: .github/workflows/pr-checks.yaml (hunk 2 above)
Comment (deuszx) [reply .github/workflows/pr-checks.yaml:28]: Changed the `all-drink` to also run `check-all`. Good eye.

File: farm/tests/src/psp22/mod.rs [hunk 3]
```diff
@@ -1,30 +1,26 @@
...
-use anyhow::Result;
-use drink::{runtime::MinimalRuntime, session::Session};
+use crate::utils::handle_ink_error;
+
+use drink::{runtime::MinimalRuntime, session::Session, AccountId32};
 use ink_primitives::AccountId;
-use ink_wrapper_types::{util::ToAccountId, Connection};
+use ink_wrapper_types::{Connection, ToAccountId};
 
 /// Uploads and creates a PSP22 instance with 1B*10^18 issuance and given names.
```
[the remark this answers is not in the document]
Comment (deuszx) [reply farm/tests/src/psp22/mod.rs]: Changed.

**Review by krzysztofziobro — APPROVED** [review]

**Pull request merged**